paper/paper.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "Speakerbox: A Workflow for Annotation and Training for Multi-Speaker Classification Models"
+tags:
+    - Python
+    - speech recognition
+    - machine learning
+authors:
+    - name: Eva Maxfield Brown
+      orcid: 0000-0003-2564-0373
+      affiliation: 1
+    - name: To Huynh
+      orcid: 0000-0002-9664-3662
+      affiliation: 2
+    - name: Isaac Na
+      orcid: 0000-0002-0182-1615
+      affiliation: 3
+    - name: Nicholas Weber
+      orcid: 0000-0002-6008-3763
+      affiliation: 1
+
+affiliations:
+    - name: University of Washington Information School, University of Washington, Seattle
+      index: 1
+    - name: University of Washington, Seattle
+      index: 2
+    - name: Washington University, St. Louis
+      index: 3
+
+date: 7 September 2022
+bibliography: paper.bib
+---
+
+# Summary
+
+
+
+# Statement of Need
+
+Speaker annotated audio and transcript data has previously been used to create comprehensive analyses of conversation dynamics [@jacobi_justice_2017;@morris_reexamining_2001;@osborn_speaking_2010;@miller_effect_2022;@maltzman_politics_1996;@slapin_sound_2020]. However, multi-speaker audio classification models for the purpose of speaker identification can be cumbersome and expensive to train and unweldy to apply. Speaker diarization, "the unsupervised identification of each speaker within an audio stream and the intervals during which each speaker is active" [@SpeakerDia2012], is a useful method in certain applications and while there are free and open tools available for speaker diarization [@Bredin2020;@Bredin2021], in many cases, unsupervised methods do not adequately meet the needs of researchers. Openly available and easy-to-use tools for supervised methods of speaker identification are lacking and Speakerbox fills this gap by providing:
```

Comment:
It reads awkward to lead with previous work. 

Suggest instead writing an opener like "Transcribing audio-recordings is an increasingly important applied research problem in Natural Language Processing. Much progress has been made on improving the fidelity of automated transcript generation, Y and Z. For example, speaker annotated audio and transcript data has previously been used to create comprehensive analyses of conversation dynamics [@jacobi_justice_2017;@morris_reexamining_2001;@osborn_speaking_2010;@miller_effect_2022;@maltzman_politics_1996;@slapin_sound_2020].